src/main/scala/org/broadinstitute/hail/stats/package.scala

```diff
@@ -2,9 +2,43 @@ package org.broadinstitute.hail
 
 import org.apache.commons.math3.distribution.HypergeometricDistribution
 import org.broadinstitute.hail.utils._
+import org.broadinstitute.hail.variant.Genotype
 
 package object stats {
 
+  def imputeInfoScore(gs: Iterable[Genotype]) = {
```

Comment:
One thing, though: if we're going to keep `imputeInfoScore`, we should have its return type specified at declaration.  This makes code much more accessible